copy_props.ps1

```diff
@@ -0,0 +1,25 @@
+###
```

Comment:
See https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help?view=powershell-7.2 for adding help to powershell scripts.

You can rewrite this comment/help section like:
```
<#
.Synopsis
Configure Sample projects build properties (Directory.Build.props).
#>
```

And then from a powershell cli run:
```
PS > get-help ./copy_props.ps1

NAME
    /Users/ccheetham/src/github.com/SteeltoeOSS/Samples/copy_props.ps1

SYNOPSIS
    Configure Sample projects build properties (Directory.Build.props).
...
```
```